aws/sign_test.go

```diff
@@ -3,15 +3,18 @@ package aws
 import (
 	"bytes"
 	"fmt"
-	. "gopkg.in/check.v1"
 	"net/http"
 	"time"
+
+	. "gopkg.in/check.v1"
 )
 
 var _ = Suite(&SigningSuite{})
 
 type SigningSuite struct{}
 
+const v4skipReason = `"The signing methodology is a "one size fits all" approach. The hashes we check against don't include headers that are added in as requisite parts for S3. That doesn't mean the tests are invalid, or that signing is broken for these examples, but as long as we're adding heads in, it's impossible to know what the new signature should be. We should revaluate these later.`
```

Comment:
How about:

```go
// TODO(katco-): The signing methodology is a "one size fits all" approach.
// The hashes we check against against don't include headers that are added
// in as requisite parts for S3. That doesn't mean the tests are invalid, or that
// signing is broken for these examples, but as long as we're adding heads in,
// it's impossible to know what the new signature should be. We should reevaluate
// these later. See issue #XYZ.
const v4SkipReason = `Extra headers present - cannot predict generated signature (issue #XYZ)`
```

This will show nicely as we're running tests (i.e. not splitting the output on multiple lines), but also gives the reader of this source enough context to understand the issue. Please, file an issue for this and replace the #XYZ with the issue number.